### PR TITLE
Ignore sourcemap comments

### LINKED ISSUE #1431: comment-word-blacklist should only ignore sourcemap comments
Ref: https://github.com/stylelint/stylelint/pull/1428#discussion_r66724370

### PR DESCRIPTION
**Pull request by m-allanson**
While working on this I realised that `comment-word-blacklist` didn't seem to be linting as advertised. blacklist strings were only triggering warnings if the comment exactly matched the string. 

This PR updates the rule to also warn when a comment _contains_ a blacklist string. 

Closes #1431

### CODE REVIEW

File: src/utils/containsString.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * matchesStringOrRegExp() util to match regexes.
+ *
+ * @param {string} input
+ * @param {string|array} comparison
+ * @return {boolean|object} `false` if no match is found.
+ *   If a match is found, returns an object with these properties:
+ *   - `match`: the `input` value that had a match
+ *   - `pattern`: the `comparison` pattern that had a match
+ */
+export default function containsString(input, comparison) {
```
Comment (evilebottnawi) [inline src/utils/containsString.js]: Be good if we added tests for this utils.

**Comment by evilebottnawi** [comment]
LGTM, one comment.

**Comment by m-allanson** [comment]
This is now updated with more tests.

**Comment by evilebottnawi** [comment]
LGTM

**Comment by jeddy3** [comment]
@m-allanson Thanks! Looks good to me.

**Pull request merged**